modules/mod_ginger_base/controllers/controller_auth.erl

```diff
@@ -48,6 +50,39 @@ process_post(Req, State = #state{mode = new}) ->
     {ok, Id} = mod_signup:signup([{email, Email}], [{identity, Identity}], RequestConfirm, Context),
     ok = mod_signup:request_verification(Id, Context),
     {{halt, 204}, Req1, State};
+process_post(Req, State = #state{mode = reset_password}) ->
+    Context = State#state.context,
+    {Body, Req1} = wrq:req_body(Req),
+    #{u := Username,
```

Comment:
`u` is a tad cryptic; why not just `username`?

Reply:
You're right, I'll adapt it. As for the why, I've basically copied and adapted this code from Zotonic's `mod_authentication`, which uses this shorthand :)

Reply:
Not this part, though. 😉